src/backend/gpopt/CGPOptimizer.cpp

```diff
@@ -272,14 +288,30 @@ TerminateGPOPT()
 {
 	GPOS_TRY
 	{
-		return CGPOptimizer::TerminateGPOPT();
```

Comment:
Do we really need to rewrite TerminateGPOPT()?

Reply:
The TerminateGPOPT function has been rewritten to match the InitGPOPT function.

Reply:
reverted